pages/common/awk.md

```diff
@@ -1,36 +1,33 @@
 # awk
 
 > A versatile programming language for working on files.
+> See also: `sed`, `ed`.
 > More information: <https://manned.org/awk.1>.
 
-- Execute an expression:
+- Execute commands:
 
-`awk "{{expression}}"`
+`awk '{{{ print gensub("apple", "mango", "g") }}}'`
 
 - Execute a script:
 
-`awk --file {{path/to/file}}`
+`awk --file {{path/to/script.awk}}`
 
 - Check a script for syntax errors:
 
-`awk --lint --file {{path/to/file}}`
+`awk --lint --file {{path/to/script.awk}}`
 
-- Execute an expression with the specified field separator:
+- Execute commands with the specified field separator:
```

Comment:
Let's be more specific here.
```suggestion
- Execute an AWK expression with the specified field separator:
```

`expression` is the most technically correct language here.